201-front-door-api-management/azuredeploy.json

```diff
@@ -0,0 +1,1048 @@
+{
```

Comment:
See the pipeline results for best practice violations and fix what's reported

Reply:
Done. I also redeployed the template to make sure that it works after all the changes I made.